swan-lake/integration-tools/asyncapi-tool.md

```
@@ -177,3 +217,149 @@ service AppService on webhookListener {
+    schemas:
+      CloseFrame:
+        x-close-frame: true
+        type: object
+        properties:
+          code:
+            type: integer
+```
+
+> **Note:** The spec must define exactly one channel.
+
+## Usage
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)